policy/templates/releng/.github/dependabot.yml

```diff
@@ -9,15 +9,15 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
-{{ if eq .Name "tyk-pump" }}
+{{ if or (eq .Name "tyk-pump") (eq .Name "tyk-sink") }}
```

Comment:
Might have to add this on top of the latest changes for tib.

Reply:
Done 👍